File: javascript/7_kyu/square_every_digit.js
// Square Every Digit
// https://www.codewars.com/kata/546e2562b03326a88e000020/train/javascript

function squareDigits(num) {
    var ans = [];
    var numbers = Array.from(String(num), Number);

    for (var i = 0; i < numbers.length; i++) {
        ans.push(numbers[i]**2);
    }

    return parseInt(ans.join(""));
}

console.log(squareDigits(9119));

File: python/7_kyu/square_every_digit.py
# Square Every Digit
# https://www.codewars.com/kata/546e2562b03326a88e000020/train/python

def square_digits(num):
    ans = []

    for i in list(str(num)):
        ans.append(str(int(i)**2))

    return int(''.join(ans))

print(square_digits(9119))